lib/livebook/runtime.ex

```diff
@@ -477,4 +477,10 @@ defprotocol Livebook.Runtime do
   """
   @spec put_system_envs(t(), list({String.t(), String.t()})) :: :ok
   def put_system_envs(runtime, secrets)
+
+  @doc """
+  Deletes Livebook secrets as environment variables
+  """
+  @spec delete_system_envs(t(), list({String.t(), String.t()})) :: :ok
+  def delete_system_envs(runtime, secrets)
```

Comment:
```suggestion
  @doc """
  Unsets the given environment variables.
  """
  @spec delete_system_envs(t(), list(String.t())) :: :ok
  def delete_system_envs(runtime, names)
```

Let's also update the wording in `put_system_envs`, we made changes back and forth and it's no longer accurate :)

Reply:
Same for the runtimes and runtime server :see_no_evil: